SimCalorimetry/HGCalSimProducers/interface/HGCDigitizerBase.h

```diff
@@ -87,6 +87,9 @@ class HGCDigitizerBase {
   
   //noise level
   std::vector<float> noise_fC_;
+
+  //charge collection efficiency
+  std::vector<double> cce_;
```

Comment:
cce_ is used as an argument in  myFEelectronics_->runShaper(), which requires a float.
No need of having  cce_ defined as double and the converted into a float, then: it can be defined float as in HGCDigitizer.h